doc/man/man3/crypto_argon2i.3monocypher

```diff
@@ -141,22 +142,22 @@ Use
 .Xr crypto_verify32 3monocypher
 or
 .Xr crypto_verify64 3monocypher
-to compare password hashes to prevent timing attacks, which are feasible
-against the weakest passwords.
+to compare password hashes to prevent timing attacks.
 .Pp
-The hardness of the computation can be chosen thus:
-.Bl -bullet
-.It
-Decide how long the computation should take.
-For user authentication, this is typically somewhere between half a
-second (convenient) and several seconds (paranoid).
-.It
-Try to hash a password with 3 iterations and 100000 blocks (one hundred
-megabytes) to establish a baseline.
+To select the nb_blocks and nb_iterations parameters, it should first be
+decided how long the computation should take.
+For user authentication, somewhere between half a second (convenient)
+and several seconds (paranoid) is a good starting point.
+The computation should use as much memory as can be spared.
+.Pp
+Since parameter selection depends on your hardware, some trial and error
+will be required in order to determine the ideal settings.
+Three iterations and 100000 blocks (that is, one hundred megabytes of
+memory) is a good starting point.
 Adjust
```

Comment:
Good change. I would remove the first instance of "is a good starting point", though: the time it takes to hash the password is not a starting point, it's the goal; and I like to avoid repetitions